packages/langchain/src/openai/util.test.ts

```diff
@@ -2,55 +2,71 @@ import {
   AzureOpenAiCreateChatCompletionResponse,
   AzureOpenAiCreateChatCompletionRequest
 } from '@sap-ai-sdk/foundation-models';
-import nock from 'nock';
-import { BaseMessage, HumanMessage } from '@langchain/core/messages';
 import {
-  mockClientCredentialsGrantCall,
-  parseMockResponse
-} from '../../../../test-util/mock-http.js';
+  BaseMessage,
+  RemoveMessage,
```

Comment:
What is that?

Reply:
RemoveMessage is another message type from langchain which is not a supported message type for openai.
I just use it to cause an error case.